paderbox/io/play.py

```diff
@@ -147,9 +153,78 @@ def _repr_html_(self):
                     </tr>
                 </table>
                 """.format(self.name, audio_html)
-        na = NamedAudio(data, rate=sample_rate, **kwargs)
-        na.name = name
-        display(na)
+        audio = NamedAudio(data, rate=sample_rate, **kwargs)
+        audio.name = name
+
+    if display:
+        IPython.display.display(audio)
+    else:
+        return audio
+
+
+class Play:
+    def __init__(self, sample_rate=None, **kwargs):
+        """
+        See `pb.io.play`. This is a wrapper around `pb.io.play`, where you can
+        change the defaults.
+
+        Args:
+            channel: Channel, if you have a multichannel stft signal or a
+                multichannel audio file.
+            sample_rate: Sampling rate in Hz.
+            size: STFT window size
+            shift: STFT shift
+            window: STFT analysis window
+            window_length: STFT window_length
+            scale: Scale the Volume, currently only amplification with clip
+                is supported.
+            name: If name is set, then in ipynb table with name and audio is
+                displayed
+            stereo: If set to true, you can listen to channel as defined by
+                `channel` parameter and the next channel at the same time.
+            normalize: It true, normalize the data to have values in the range
+                from 0 to 1. Can only be disabled with newer IPython versions.
+
+        Returns:
+
+        """
+        self.kwargs = {
+            **kwargs,
+        }
```

Comment:
Why copy kwargs?

Reply:
Good point, in the initial version `sample_rate` had a default argument and was added here.